apps/cookbook/src/app/examples/avatar-example/examples/image-error.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-avatar-example-image-error',
+  template: `<kirby-avatar
+  [imageSrc]="avatarSrc"
+  (imageError)="showFallbackImageOnError($event)"
+  size="lg"
+></kirby-avatar>`,
+  ts: `avatarSrc: string = 'bad-image-url.png';
+
+showFallbackImageOnError(event: ErrorEvent) {
+  const fallbackImageSrc =
+                'assets/images/avatar-not-found.png';
+  this.avatarSrc = fallbackImageSrc;
+  // Possibly do something with the ErrorEvent parameter...
+}
+  `,
```

Comment:
Trailing linebreaks will show up in the code snippet as it's pre-formatted:
![image](https://github.com/user-attachments/assets/afbc322c-b50f-41da-aa99-772ab48329e5)
```suggestion
}`,
```